python/cugraph-dgl/examples/dataset_from_disk_cudf.ipynb

```diff
@@ -95,27 +105,26 @@
     "g, train_idx = load_dgl_dataset()\n",
     "g = cugraph_dgl.cugraph_storage_from_heterograph(g, single_gpu=single_gpu)\n",
     "\n",
-    "temp_dir = tempfile.TemporaryDirectory()\n",
     "batch_size = 1024\n",
     "fanout_vals=[25, 25]\n",
     "sampler = cugraph_dgl.dataloading.NeighborSampler(fanout_vals)\n",
     "dataloader = cugraph_dgl.dataloading.DataLoader(\n",
     "    g,                               \n",
     "    train_idx.to('cuda'),                        # train_nid must be on GPU.\n",
     "    sampler,\n",
-    "    sampling_output_dir=temp_dir.name, # Path to save sampling results to\n",
+    "    sampling_output_dir=\"/raid/vjawa/obgn_products_sampling/\", # Path to save sampling results to\n",
```

Comment:
should this go back to using a proper tempdir instead of hardcoding yours?  Or if not, should there be a comment explaining that users have to edit this path accordingly?